examples/README.md

```diff
@@ -7,7 +7,13 @@ Each folder contains three files:
 1. **openapi_swagger.json**:  API producer defined [OpenAPI Specification](https://github.com/OAI/OpenAPI-Specification).
 
 2. **service_config_generated.json**: Service configuration generated by Google Service Management.
-To generate this, deploy `openapi_swagger.json` to service management and run:
+To generate this, deploy `openapi_swagger.json` to service management:
+
+```shell script
+gcloud endpoints services deploy ./examples/$DIRECTORY/openapi_swagger.json
+```
+
+Then run:
 
 ```shell script
 gcloud endpoints configs describe "${CONFIG_ID}" \
```

Comment:
We may have to use "curl" to fetch service config, just like `gcloud_build_image`. 
otherwise, the proto descriptor is missing, and grpc_transcoder is not added.

Reply:
Right, updated README with this. I'll make a PR later to cleanup instructions for grpc vs openapi.